src/main/java/wooteco/subway/ui/StationController.java

```diff
@@ -1,39 +1,54 @@
 package wooteco.subway.ui;
 
+import java.net.URI;
+import java.util.List;
+import java.util.stream.Collectors;
+import javax.validation.Valid;
 import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.*;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
 import wooteco.subway.dao.StationDao;
 import wooteco.subway.domain.Station;
 import wooteco.subway.dto.StationRequest;
 import wooteco.subway.dto.StationResponse;
 
-import java.net.URI;
-import java.util.List;
-import java.util.stream.Collectors;
-
 @RestController
+@RequestMapping("/stations")
 public class StationController {
 
-    @PostMapping("/stations")
-    public ResponseEntity<StationResponse> createStation(@RequestBody StationRequest stationRequest) {
-        Station station = new Station(stationRequest.getName());
-        Station newStation = StationDao.save(station);
-        StationResponse stationResponse = new StationResponse(newStation.getId(), newStation.getName());
+    private final StationDao stationDao;
+
+    public StationController(StationDao stationDao) {
+        this.stationDao = stationDao;
+    }
+
+    @PostMapping
+    public ResponseEntity<StationResponse> createStation(@RequestBody @Valid StationRequest stationRequest) {
+        Station station = stationRequest.toEntity();
+        Station newStation = stationDao.save(station);
+        StationResponse stationResponse = new StationResponse(station);
         return ResponseEntity.created(URI.create("/stations/" + newStation.getId())).body(stationResponse);
```

Comment:
(제 생각에는) 체이닝 메서드의 경우 적절하게 개행을 하면 더 가독성이 좋아지는 것 같아요 :)

> 물론 이렇게 한 라인으로 쓰는 것이 더 가독성이 좋다고 생각하는 사람들도 많아서 이 부분은 조시의 생각대로 가져가도 됩니다

```suggestion
        return ResponseEntity
                .created(URI.create("/stations/" + newStation.getId()))
                .body(stationResponse);
```